### ISSUE TITLE
Opam ubuntu ppa doesn't contain focal distribution while opam 2.0 documentation says "Ubuntu 18.04 and newer"

### ISSUE THREAD

**Issue by MercierCorentin** [ocaml/opam#4254]
I recently installed Ubuntu 20.04 focal expecting to check opam install. After adding opam ubuntu ppa I get this error:
```
...
Err:8 http://ppa.launchpad.net/avsm/ppa/ubuntu focal Release                   
  404  Not Found [IP: 91.189.95.83 80]
...
E: The repository 'http://ppa.launchpad.net/avsm/ppa/ubuntu focal Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
Is this behavior expected ? If yes it would be useful to have a smaller Ubuntu version range in opam's 2.0 documentation.
If you are currently working on it, I didn't see it but might have missed this piece of information.

**Comment by rjbou** [comment]
I'll ping @avsm on this one.

**Comment by avsm** [comment]
opam 2.0.5 is available in Focal by default, so the documentation can be tightened to reduce the version range.

**Comment by dra27** [comment]
I think this is all fixed - opam 2.1.0~rc2 is also in the ppa for bionic, focal, groovy and hirsute (thanks, @avsm!)
